firebase-appdistribution/test-app/src/main/java/com/googletest/firebase/appdistribution/testapp/NotificationFeedbackTrigger.kt

```diff
@@ -13,33 +13,34 @@ import android.os.Build
 import android.os.Bundle
 import android.util.Log
 import androidx.activity.result.ActivityResultCaller
-import androidx.activity.result.ActivityResultLauncher
 import androidx.activity.result.contract.ActivityResultContracts
-import androidx.appcompat.app.AppCompatActivity
+import androidx.annotation.RequiresApi
 import androidx.core.app.NotificationCompat
 import androidx.core.app.NotificationManagerCompat
 import androidx.core.content.ContextCompat
 import com.google.firebase.appdistribution.ktx.appDistribution
 import com.google.firebase.ktx.Firebase
-import com.googletest.firebase.appdistribution.testapp.NotificationFeedbackTrigger.SCREENSHOT_FILE_NAME
-import com.googletest.firebase.appdistribution.testapp.NotificationFeedbackTrigger.takeScreenshot
 import java.io.IOException
-import java.util.*
 
 @SuppressLint("StaticFieldLeak") // Reference to Activity is set to null in onActivityDestroyed
 object NotificationFeedbackTrigger : Application.ActivityLifecycleCallbacks {
   private const val TAG: String = "NotificationFeedbackTrigger"
   private const val FEEBACK_NOTIFICATION_CHANNEL_ID = "InAppFeedbackNotification"
   private const val FEEDBACK_NOTIFICATION_ID = 1
-  const val SCREENSHOT_FILE_NAME = "com.googletest.firebase.appdistribution.testapp.screenshot.png"
 
   private var isEnabled = false
   private var hasRequestedPermission = false
-  private var currentActivity: Activity? = null
-  // TODO(lkellogg): this is getting too complex - simplify it by, for example, only enabling this
-  //   trigger on a per-activity basis instead of app-wide
-  private var requestPermissionLaunchers: WeakHashMap<Activity, ActivityResultLauncher<String>?> = WeakHashMap()
 
+  internal var currentActivity: Activity? = null // Activity to be used for screenshot
```

Comment:
Should we call this something like `activityToScreenshot`? Could then also remove the comment below that this "points to the previous activity" since it wouldn't be confusing that it's called "current".

Reply:
Done.